test/external/TransactionForwarder_v1.t.sol

```diff
@@ -34,7 +34,7 @@ contract TransactionForwarderV1Test is Test {
         uint signerPrivateKey
     ) public {
         // Restrict the signerKey to a space where it still should work
-        signerPrivateKey = bound(signerPrivateKey, 1, 2 ^ 128);
+        signerPrivateKey = bound(signerPrivateKey, 1, 2 ** 128);
```

Comment:
why this change? was it the formatter?

Reply:
yeah, it's a good find (even if it's just within a test). could you put that into its own PR if possible? just to keep it clean. I wouldn't be too annoyed though if you'd prefer to just keep it here :D

Reply:
Tagging @FHieser here as well, as I think he added the symbol :)